Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -333,30 +333,10 @@ if ($subKeys) {
             foreach ($m in $Matches.Values) {
                 $foundDll = (Get-Item "HKLM:\SOFTWARE\Classes\ClSid\{$m}\InprocServer32" -ErrorAction SilentlyContinue).GetValue("").trim('"')
                 if ($null -eq $foundDll) {
-                    Write-Host "No AMSI Dlls was found for $m, possible AMSI misconfiguration" -ForegroundColor Red
+                    Write-Host "No AMSI Dlls were found for $m, possible AMSI misconfiguration" -ForegroundColor Red
                 } else {
                     Write-Verbose "AMSI $m was found"
                     $AMSIDll.Add($foundDll)
-                    # cSpell:ignore MPCLIENT
```

Comment:
@DKhrebin You ended up removing this causing the spell check failure again. 

Please add this back and also squash your commits down to 1 because there is a lot of add/removal in this simple PR.